src/components/StickyPlayer.vue

```diff
@@ -40,7 +42,8 @@ export default {
   methods: {
     ...mapActions([
       'updatePlayerState',
-      'playEpisode'
+      'playEpisode',
+      'playEpisodeEvent'
     ]),
     onFinished () {
```

Comment:
I'd like to add a `completedEpisode` event here

Reply:
That'll make a lot of sense